### PR TITLE
Add SUITE_CONTEXT property, 'revoked' export tests.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
See also corresonding PR https://github.com/digitalbazaar/crypto-ld/pull/58

### CODE REVIEW

File: lib/X25519KeyAgreementKey2019.js [hunk 1]
```diff
@@ -226,15 +232,20 @@ export class X25519KeyAgreementKey2019 extends LDKeyPair {
     }
     const exportedKey = {
```
Comment (dlongley) [inline lib/X25519KeyAgreementKey2019.js:233]: Noticed this is different from ed25519 suite ... no `@context` (and if we are adding that here, we probably need that `includeContext = false` param).

Comment (dmitrizagidulin) [reply lib/X25519KeyAgreementKey2019.js:233]: Right, so, we don't actually have a standalone context for the X 2019 keys.

Comment (mattcollier) [reply lib/X25519KeyAgreementKey2019.js:233]: should we still have the `includeContext` flag and include some appropriate context?

File: test/X25519KeyAgreementKey2019.spec.js [hunk 2]
```diff
@@ -142,12 +142,17 @@ describe('X25519KeyAgreementKey2019', () => {
...
+        controller: 'did:example:1234'
+      });
+      key.revoked = '2019-10-12T07:20:50.52Z';
 
       const exported = key.export({publicKey: true, privateKey: true});
-      expect(exported).to.have.keys([
-        'id', 'controller', 'type', 'publicKeyBase58', 'privateKeyBase58'
-      ]);
+      expect(exported).to.have.keys(['id', 'type', 'controller',
+        'publicKeyBase58', 'privateKeyBase58', 'revoked']);
```
Comment (mattcollier) [inline test/X25519KeyAgreementKey2019.spec.js:152]: previous assertion exhibited the proper style here.

File: test/X25519KeyAgreementKey2019.spec.js [hunk 3]
```diff
@@ -142,12 +142,17 @@ describe('X25519KeyAgreementKey2019', () => {
     });
 
     it('should export both public and private key', async () => {
-      const key = await X25519KeyAgreementKey2019.generate();
+      const key = await X25519KeyAgreementKey2019.generate({
+        controller: 'did:example:1234'
+      });
+      key.revoked = '2019-10-12T07:20:50.52Z';
```
Comment (mattcollier) [inline test/X25519KeyAgreementKey2019.spec.js:148]: use `pastDate` one liner.

**Review by mattcollier — CHANGES_REQUESTED** [review]
style issues and open question about `includeContext`.

File: test/X25519KeyAgreementKey2019.spec.js (hunk 3 above)
Comment (dmitrizagidulin) [reply test/X25519KeyAgreementKey2019.spec.js]: Changed.

File: test/X25519KeyAgreementKey2019.spec.js (hunk 2 above)
Comment (dmitrizagidulin) [reply test/X25519KeyAgreementKey2019.spec.js]: Updated.

File: lib/X25519KeyAgreementKey2019.js (hunk 1 above)
Comment (dmitrizagidulin) [reply lib/X25519KeyAgreementKey2019.js:234]: Added.

**Review by mattcollier — APPROVED** [review]

**Pull request merged**